internal/test/e2e/run.go

```diff
@@ -411,6 +417,34 @@ func splitTinkerbellTests(awsSession *session.Session, testsList []string, conf
 	return runConfs, nil
 }
 
+func runAirgappedTinkerbellTests(awsSession *session.Session, testsList []string, conf ParallelRunConf, testRunnerConfig *TestInfraConfig, runConfs []instanceRunConf) ([]instanceRunConf, error) {
+	tinkerbellTests := getTinkerbellAirgappedTests(testsList)
+	if len(tinkerbellTests) == 0 {
+		conf.Logger.V(1).Info("No tinkerbell airgapped test to run")
+		return runConfs, nil
+	}
+
+	conf.Logger.V(1).Info("INFO:", "tinkerbellAirGappedTests", len(tinkerbellTests))
+
+	err := s3.DownloadToDisk(awsSession, os.Getenv(tinkerbellAirgappedHardwareS3FileKeyEnvVar), conf.StorageBucket, e2eAirgappedHardwareCsvFilePath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to download tinkerbell airgapped hardware csv: %v", err)
```

Comment:
nit: i think we are avoiding "failed to xxx" for all error statement

Reply:
I don't mind changing the error statement, but I'm not sure which is our preferred alternative. And this error message is not user facing, I think it should be fine.

Reply:
```suggestion
		return nil, fmt.Errorf("downloading tinkerbell airgapped hardware csv: %v", err)
```